app/helpers/placement_group.rb

```diff
@@ -1,3 +0,0 @@
-module PlacementGroupHelper
-  include_concern 'TextualSummary'
-end
```

Comment:
compare to the correctly named one: https://github.com/ManageIQ/manageiq-ui-classic/blob/1d212dd1942d7f0e563d46eacf4c152156aaf823/app/helpers/placement_group_helper.rb